### PR TITLE
*: Add TiDB Dashboard session docs

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <me@breeswish.org>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add TiDB Dashboard docs about sessions.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions) Version number needs change
- [x] v5.0 (TiDB 5.0 versions) Version number needs change
- [x] v4.0 (TiDB 4.0 versions) Version number needs change
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM except for https://github.com/pingcap/docs-cn/pull/6711/files#r674572235

File: dashboard/dashboard-session-sso.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+summary: 了解如何配置 TiDB Dashboard 启用 SSO 登录。
+---
+
+# 配置 TiDB Dashboard 使用 SSO 登录
+
+TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点登录 (Single Sign-On)。配置 TiDB Dashboard 启用 SSO 登录后，你可以通过配置的 SSO 服务进行登录鉴权，无需输入 SQL 用户名和密码即可登录到 TiDB Dashboard。
+
+> **注意：**
+>
+> 该功能仅在 v4.0.14 或更高版本的集群中可用。
```
Comment (TomShawn) [inline dashboard/dashboard-session-sso.md:12]: 这个需要现在 cherry-pick 到 5.1 和 5.0 吗？4.0.14 是晚于 5.1.0 和 5.0.3 的吧。5.1.0 和 5.0.3 之前的版本能使用这个功能吗？

Comment (breeswish) [reply dashboard/dashboard-session-sso.md:12]: 5.0.x 和 5.1.x 的文档需要另外写，例如改成 v5.1.1 可用

File: dashboard/dashboard-session-share.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+   ![操作示例](/media/dashboard/dashboard-session-share-settings-1.png)
+
+   > **注意：**
+   >
+   > 出于安全考虑，已分享的会话中不能使用分享功能将该会话再次分享给其他人。
+
+4. 在弹出的对话框中，可以对分享进行细节配置：
+
+   - 有效时间：分享的会话在多少时间内有效。当前会话的登出不影响已分享会话的有效时间。
```
[the remark this answers is not in the document]
Comment (breeswish) [reply dashboard/dashboard-session-share.md:30]: 会话 A 分享得到会话 B，然后登出会话 A，此时会话 B 会保持有效

[the remark this answers is not in the document]
Comment (TomShawn) [reply dashboard/dashboard-session-share.md:30]:
```suggestion
   - 有效时间：分享的会话在多少时间内有效。登出当前会话不影响已分享会话的有效时间。
```

File: dashboard/dashboard-session-sso.md (hunk 1 above)
Comment (TomShawn) [reply dashboard/dashboard-session-sso.md:12]: 那就是 cherry-pick 到 release-5.0 和 release-5.1 暂时不合了。

File: dashboard/dashboard-session-share.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+# 分享 TiDB Dashboard 会话
+
+你可以将当前的 TiDB Dashboard 会话安全地分享给其他用户访问，这样其他用户无需要知道登录账号密码即可访问 TiDB Dashboard 并进行操作。
+
+> 注意：
+>
+> 该功能仅在 v4.0.6 或更高版本的集群中可用。
+
```
Comment (TomShawn) [inline dashboard/dashboard-session-share.md:13]:
```suggestion
```

File: dashboard/dashboard-session-share.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+   >
+   > 出于安全考虑，已分享的会话中不能使用分享功能将该会话再次分享给其他人。
+
+4. 在弹出的对话框中，对分享进行细节配置：
+
+   - 有效时间：分享的会话在多少时间内有效。登出当前会话不影响已分享会话的有效时间。
+
+   - 以只读权限分享：分享的会话为只读，例如不允许进行配置修改等操作。
+
+   备注：“以只读权限分享”功能仅在 v4.0.14 或更高版本中可用。
```
Comment (TomShawn) [inline dashboard/dashboard-session-share.md:34]:
```suggestion
```

**Comment by breeswish** [comment]
@shhdgit PTAL, thanks!

File: dashboard/dashboard-session-share.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+   >
+   > 出于安全考虑，已分享的会话中不能使用分享功能将该会话再次分享给其他人。
+
+4. 在弹出的对话框中，对分享进行细节配置：
+
+   - 有效时间：分享的会话在多少时间内有效。登出当前会话不影响已分享会话的有效时间。
+
+   - 以只读权限分享：分享的会话为只读，例如不允许进行配置修改等操作。
+
+
```
Comment (TomShawn) [inline dashboard/dashboard-session-share.md:30]:
```suggestion
```

File: dashboard/dashboard-session-share.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+   > **注意：**
+   >
+   > 出于安全考虑，已分享的会话中不能使用分享功能将该会话再次分享给其他人。
+
+4. 在弹出的对话框中，对分享进行细节配置：
+
+   - 有效时间：分享的会话在多少时间内有效。登出当前会话不影响已分享会话的有效时间。
+
+   - 以只读权限分享：分享的会话为只读，例如不允许进行配置修改等操作。
```
Comment (shhdgit) [inline dashboard/dashboard-session-share.md:28]: 这个目前是怎么做的呀

Comment (breeswish) [reply dashboard/dashboard-session-share.md:28]: TiDB Dashboard 侧限制的

Comment (shhdgit) [reply dashboard/dashboard-session-share.md:28]: 看到了

**Comment by shhdgit** [comment]
LGTM.

File: dashboard/dashboard-session-sso.md (hunk 1 above)
Comment (TomShawn) [reply dashboard/dashboard-session-sso.md]: 已在 https://github.com/pingcap/docs-cn/pull/6728 中加回 4.0 的版本说明

**Pull request merged**